firefly_client/firefly_client.py

```diff
@@ -575,7 +604,7 @@ def dispatch_remote_action_by_post(self, channel, action_type, payload):
         """
 
         action = {'type': action_type, 'payload': payload}
-        data = {'channelID': channel, 'cmd': 'pushAction', 'action': json.dumps(action)}
+        data = {'channelID': self.channel, 'cmd': 'pushAction', 'action': json.dumps(action)}
```

Comment:
It would be nice if we could use the similar approach to set 'renderTreeId' in action payload here. (Rather than setting it in the specific payloads.)
```
if self.render_tree_id:
            payload['renderTreeId']= self.render_tree_id
```
Can it be just ignored if not relevant for a particular action?